__tests__/app.test.js

```diff
@@ -78,4 +78,31 @@ describe("app", () => {
         });
     });
   });
+  describe("/api/reviews/:review_id", () => {
+    it("200: it should respond with a review object with the correct properties", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          const { review } = body;
+          expect(review).toHaveProperty("review_id");
+          expect(review).toHaveProperty("title");
+          expect(review).toHaveProperty("review_body");
+          expect(review).toHaveProperty("designer");
+          expect(review).toHaveProperty("review_img_url");
+          expect(review).toHaveProperty("votes");
+          expect(review).toHaveProperty("category");
+          expect(review).toHaveProperty("owner");
+          expect(review).toHaveProperty("created_at");
+        });
```

Comment:
Rather than checking the properties that are coming back, we should check to see what data types are coming back. Take a look at toMatchObject and then use things like   'title: expect.any(String)' 👍